docs/config/QUnit.extend.md

```diff
@@ -1,9 +1,13 @@
 ---
 layout: default
-categories: [config]
+categories:
+- config
+- deprecated
 title: QUnit.extend()
+status: deprecated
 description: Copy the properties defined by the <code>mixin</code> object into the <code>target</code> object.
 version_added: "1.0"
+version_deprecated: "2.12"
```

Comment:
I've only recently added support for these version pairs, and only once before a release. The way I did it that one time is setting it to [unreleased](https://github.com/qunitjs/qunit/search?q=unreleased) (since it'll go live on the main site as soon I land this PR), and then as part of the release prep commit, replace with the version.

I'm not worried about it being unknown which version it'll end up in, I'm sure it'll end up in 2.12. But I want to avoid broken links or it appearing as if it's already in effect when it isn't yet.

Reply:
Actually, I'll do a release today so I'm landing this as-is for now.